src/hotspot/share/cds/filemap.cpp

```diff
@@ -361,7 +361,6 @@ void SharedClassPathEntry::set_name(const char* name, TRAPS) {
 }
 
 void SharedClassPathEntry::copy_from(SharedClassPathEntry* ent, ClassLoaderData* loader_data, TRAPS) {
-  assert(ent != NULL, "sanity");
```

Comment:
This removal seems to be unrelated to this PR.

Reply:
Good catch, thanks.
The change is not part of mines. I think, it came up after merging with master. 
The line is back to the code now.